docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -448,4 +449,18 @@ If you don't specify a `preSubmitInfo` section, no notice or checkbox appears ab
 
 For the code implementation, see [`PreSubmitSection`](../../src/js/components/PreSubmitSection.jsx) and [`SubmitController`](../../src/js/review/SubmitController.jsx).
 
+### Remote error and event monitoring
+
+If you provide a function for `formConfig.recordEvent`, the library calls that function when notable events occur. If you do not provide a function, the library logs these events onto the browser console.
+
+The `recordEvent` function receives a single object that can contain different information depending on the kind of event being reported. Events reported by the library always have a `event` property that is a string describing the event. The events currently supported by the library are:
+* **validation-failed-on-submit**: The user completed the form and tried to submit it, but there were still validation errors. This is most likely an error in the form definition or one of the React components, since the form should already be fully validated by the time this point is reached.
```

Comment:
> This is most likely an error in the form definition or one of the React components, since the form should already be fully validated by the time this point is reached.

Can we get away with:

"This is likely an error in the form definition or a React component, since the form should be completely validated at this point."